pkg/constants/indices.go

```diff
@@ -2,6 +2,7 @@ package constants
 
 const (
 	IndexByPhysicalName  = "IndexByPhysicalName"
+	IndexByVirtualName   = "IndexByVirtualName"
```

Comment:
CSIStorageCapacity is namespaced and we sync from Host->Virtual, so we need to look up by virtual name